pkgs/development/tools/rust/cargo-toml-lint/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib
+, rustPlatform
+, fetchCrate
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-toml-lint";
+  version = "0.1.1";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-U3y9gnFvkqJmyFqRAUQorJQY0iRzAE9UUXzFmgZIyaM=";
```

Comment:
It's not everywhere yet, but I think we want new derivations to use `hash =` instead of `sha256 =`.